### PR TITLE
48: structure docs

### PR DESCRIPTION
**Pull request by magdasalatka**
Move high-level docs to wiki.
Add detailed description on this package. Describe cli tool.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,276 +1,129 @@
...
 
-* Convert existing data to RDF. There is built-in support for CSV and JSON. Transformation is done in a streaming way when possible and thus can handle very large input data.
-* Shell & pipe support: Plug to existing command-line utilities you are comfortable with and work with their output.
-* Read from and write to SPARQL endpoints
 
-barnard59 can be easily extended by providing additional functionality as plugin. We also work on user interfaces, so you do not have to write your pipelines in Turtle/RDF.
+`barnard59` expects a pipeline to be defined as linked-data. The supported formats are:
+* text/turtle
+* application/ld+json
+* ...
```
Comment (magdasalatka) [inline README.md]: @bergos can you add all supported formats?

File: README.md [hunk 2]
```diff
@@ -1,276 +1,129 @@
...
 
-We provide a list of all operations supported by barnard59 in our [primer](https://github.com/zazuko/barnard59/wiki/primer) in the wiki.
+To learn more about building your own pipeline, go [here]().
 
-## Concept
+## Requirements
+To execute pipeline with `barnard59`, you will need `node.js` environment. The required versions are ... for `node.js` and ... for `npm`.
```
Comment (magdasalatka) [inline README.md]: @bergos what are `node` and `npm` requirements?
Also, can we add it to `package.json` ?

File: README.md (hunk 1 above)
Comment (bergos) [inline README.md]:
```suggestion
* `application/ld+json`
* `application/n-triples`
* `text/turtle`
* `application/rdf+xml`
```

From [@rdfjs/formats-common](https://github.com/rdfjs-base/formats-common/blob/master/index.js#L13-L19) without the Quads formats.

File: README.md (hunk 2 above)
Comment (bergos) [reply README.md]: If not explicitly mentioned (in `package.json`), all node and npm versions which are actively maintained are supported. I'm not a fan of adding it to the `package.json`as it's just a redundant list, but :+1 for adding the textual description to the readme.

**Pull request merged**